tests/test_light.py

```diff
@@ -837,6 +838,8 @@ async def test_zha_group_light_entity(
     )
 
     assert bool(entity.state["on"]) is False
+    assert bool(entity.state["off_with_transition"]) is False
```

Comment:
`off_with_transition is False` is also tested here, but it's `False` by default anyway, even if not restored.
If we'd want to fully verify that it also restores correctly too, we'd need to modify the test further (or split it out into a separate one), as the code below expects the `on` cluster command on the `OnOff` cluster. That doesn't happen if `off_with_transition` is `True` though.
Still, the modified test checks that `off_with_transition` stays `False`. (and the `off_brightness` check is fully correct and would fail if it weren't restored. It would be `None` before this PR.)